pkg/frontend/config.go

```diff
@@ -13,12 +13,12 @@ import (
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
 
+	"github.com/grafana/dskit/netutil"
```

Comment:
Hi, linter fails due to wrong order, it should go to line 13. You can run `make lint` to verify.

```
diff --git a/pkg/frontend/config.go b/pkg/frontend/config.go
index c84acec1d..e3ba77516 100644
--- a/pkg/frontend/config.go
+++ b/pkg/frontend/config.go
@@ -10,10 +10,10 @@ import (
        "net/http"
 
        "github.com/go-kit/log"
+       "github.com/grafana/dskit/netutil"
        "github.com/pkg/errors"
        "github.com/prometheus/client_golang/prometheus"
 
-       "github.com/grafana/dskit/netutil"
        "github.com/grafana/mimir/pkg/frontend/querymiddleware"
        "github.com/grafana/mimir/pkg/frontend/transport"
        v1 "github.com/grafana/mimir/pkg/frontend/v1"
```